src/main/java/com/ghm/giftcardfleamarket/brand/mapper/BrandMapper.java

```diff
@@ -12,5 +12,5 @@ public interface BrandMapper {
 
 	List<Brand> selectBrandsByCategory(Map<String, Object> map);
 
-	String selectBrandNameByCategory(Long brandId);
+	String selectBrandNameByBrandId(Long brandId);
```

Comment:
이런 경우엔 파라미터로 유추 할수 있으니 아래처럼 메소드 시그니처를 만들면 어떨지 싶습니다.
```suggestion
	String selectBrandName(Long brandId);
```

Reply:
selectBrandName으로 네이밍 수정하겠습니다!